docs/maintenance/roreviewapi.md

```diff
@@ -1,120 +0,0 @@
-
-# roreviewapi
```

Comment:
add redirection as this will create a broken link?

Reply:
Did that in the only previous link to there.